docs/source/usage.rst

```diff
@@ -58,26 +78,43 @@ The currently supported export formats are:
   ``--post serve`` on the command-line. The ``serve`` post-processor proxies
   Reveal.js requests to a CDN if no local Reveal.js library is present.
   To make slides that don't require an internet connection, just place the
-  Reveal.js library in the same directory where your_talk.slides.html is located,
-  or point to another directory using the ``--reveal-prefix`` alias.
+  Reveal.js library in the same directory where your_talk.slides.html is
+  located, or point to another directory using the ``--reveal-prefix`` alias.
 
+Markdown
+~~~~~~~~
 * ``--to markdown``
 
   Simple markdown output.  Markdown cells are unaffected,
   and code cells indented 4 spaces.
 
+reStructuredText
+~~~~~~~~~~~~~~~~
 * ``--to rst``
 
-  Basic reStructuredText output. Useful as a starting point for embedding notebooks
-  in Sphinx docs.
+  Basic reStructuredText output. Useful as a starting point for embedding
+  notebooks in Sphinx docs.
+  
+  .. note::
 
+    nbconvert uses pandoc_ to convert between various markup languages,
+    so pandoc is a dependency when converting to latex or reStructuredText.
+
+Executable script
+~~~~~~~~~~~~~~~~~
 * ``--to script``
 
   Convert a notebook to an executable script.
-  This is the simplest way to get a Python (or other language, depending on the kernel) script out of a notebook.
-  If there were any magics in an Jupyter notebook, this may only be executable from
-  an Jupyter session.
-
+  This is the simplest way to get a Python (or other language, depending on
+  the kernel) script out of a notebook. If there were any magics in an
+  Jupyter notebook, this may only be executable from a Jupyter session.
+  
+  For example, to convert a Julia notebook to a Julia executable script::
+      
+      jupyter nbconvert --to script my_julia_notebook.ipynb my_julia_script.jl
```

Comment:
We shouldn't need to pass the name of the output file (and I'm not sure this is the syntax if we do). It should pick up the file extension automatically from the kernel.